apple/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+sudo /usr/sbin/apt-cacher-ng -c /etc/apt-cacher-ng &
```

Comment:
`sudo --non-interactive` please.

Reply:
Isnt it all non interactive if Debian is non interactive?

https://github.com/Whonix/derivative-maker/blob/1ef0e8a511f81172e59357328505c16963fdb2e5/apple/Dockerfile#L3